packages/forklift-console-plugin/src/modules/Providers/views/create/components/VSphereProviderCreateForm.tsx

```diff
@@ -65,33 +72,31 @@ export const VSphereProviderCreateForm: React.FC<VSphereProviderCreateFormProps>
       <div className="forklift--create-provider-field-error-validation">
         <Trans t={t} ns="plugin__forklift-console-plugin">
           Error: The format of the provided VDDK init image is invalid. Ensure the path is a valid
-          container image path. For example: {'<strong>'}quay.io/kubev2v/vddk:latest{'</strong>'}.
+          container image path. For example: <strong>quay.io/kubev2v/vddk:latest</strong>.
         </Trans>
       </div>
     ),
     success: (
       <div className="forklift--create-provider-field-success-validation">
         <Trans t={t} ns="plugin__forklift-console-plugin">
           Virtual Disk Development Kit (VDDK) container init image path. The path must be empty or a
-          valid container image path. For example: {'<strong>'}quay.io/kubev2v/vddk:latest
-          {'</strong>'}.
+          valid container image path. For example: <strong>quay.io/kubev2v/vddk:latest</strong>.
         </Trans>
       </div>
     ),
     default: (
       <div className="forklift--create-provider-field-default-validation">
         <Trans t={t} ns="plugin__forklift-console-plugin">
           Virtual Disk Development Kit (VDDK) container init image path. The path must be empty or a
-          valid container image path. For example: {'<strong>'}quay.io/kubev2v/vddk:latest
-          {'</strong>'}.
+          valid container image path. For example: <strong>quay.io/kubev2v/vddk:latest</strong>.
         </Trans>
       </div>
     ),
   };
 
   const vddkHelperTextPopover = (
     <Trans t={t} ns="plugin__forklift-console-plugin">
-      {'Note: It is strongly recommended to specify a VDDK init image to accelerate migrations.'}
+      Note: It is strongly recommended to specify a VDDK init image to accelerate migrations.
```

Comment:
```suggestion
     VDDK init image is optional, but it is strongly recommended to specify a VDDK init image to accelerate migrations.
```

Reply:
In a popup I like to add the note prefix to indicate that this popup includes an info type message and not a warning/error. 
The `optional` is reminded as part of the text info "The path must be empty or..."

But will change if you think it's unnecessary.

Reply:
yes, I do not like the `Note:` without some context to have the note about